configs/python/backend/backend.yaml

```diff
@@ -327,10 +327,13 @@ scanners:
           - 'type_is_tiff'
           - 'image/x-ms-bmp'
           - 'bmp_file'
+          - 'application/pdf'
+          - 'pdf_file'
       priority: 5
       options:
         extract_text: False
         tmp_directory: '/dev/shm/'
+        pdf_to_png: True
```

Comment:
IIRC you need to update `scanners.yaml` too?

Reply:
I don't think I do. That file doesn't seem to get used in our build. I can make them the same though.

Reply:
added

Reply:
Looks like it's the other way around?
https://github.com/sublime-security/strelka/blob/439d1557422ed57d6885de97c6aa17e3eabfe869/build/python/backend/Dockerfile#L151

Reply:
hmph - good catch